docs/resources/repository_conf_analysis.md

```diff
@@ -23,6 +23,7 @@ resource "cyral_repository_conf_analysis" "all_conf_analysis_enabled" {
   disable_filter_analysis = false
   enable_dataset_rewrites = true
   enable_data_masking = true
+  mask_all_occurrences = true
```

Comment:
Please make this change in the file `examples/resources/cyral_repository_conf_analysis/resource.tf` and then run:

```
make docker-compose/docs
pre-commit run --show-diff-on-failure --color=always --all-files
```

You may have to rerun `pre-commit` to fix linter issues in case they show up.